src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -64,35 +66,87 @@ public FindCommand parse(String args) throws ParseException {
+            if (licenceKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_LICENCE_PLATE + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_NRIC).isPresent()) {
             nricKeyword = argMultimap.getValue(PREFIX_NRIC).get();
+            if (nricKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_NRIC + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_PHONE).isPresent()) {
             phoneKeyword = argMultimap.getValue(PREFIX_PHONE).get();
+            if (phoneKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_PHONE + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_POLICY_NUMBER).isPresent()) {
             policyNumberKeyword = argMultimap.getValue(PREFIX_POLICY_NUMBER).get();
+            if (policyNumberKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_POLICY_NUMBER + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_TAG).isPresent()) {
             tagKeyword = argMultimap.getValue(PREFIX_TAG).get();
+            if (tagKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_TAG + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_POLICY_EXPIRY_DATE).isPresent()) {
             policyExpiryKeyword = argMultimap.getValue(PREFIX_POLICY_EXPIRY_DATE).get();
+            if (policyExpiryKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_POLICY_EXPIRY_DATE + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_EMAIL).isPresent()) {
             emailKeyword = argMultimap.getValue(PREFIX_EMAIL).get();
+            if (emailKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_EMAIL + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_POLICY_ISSUE_DATE).isPresent()) {
             policyIssueKeyword = argMultimap.getValue(PREFIX_POLICY_ISSUE_DATE).get();
+            if (policyIssueKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_POLICY_ISSUE_DATE + " ";
+            }
+            atLeastOneFieldPresent = true;
         }
         if (argMultimap.getValue(PREFIX_COMPANY).isPresent()) {
             companyKeyword = argMultimap.getValue(PREFIX_COMPANY).get();
+            if (companyKeyword.isEmpty()) {
+                emptyPrefixes += PREFIX_COMPANY + " ";
+            }
+            atLeastOneFieldPresent = true;
+        }
+
+        if (!atLeastOneFieldPresent) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        }
+
+        if (!emptyPrefixes.isEmpty()) {
```

Comment:
This looks like handling the error if a user specify a prefix, but no values. Nicely done!